src/buildblock/GeometryBlocksOnCylindrical.cxx

```diff
@@ -39,11 +39,14 @@ limitations under the License.
 #include <iostream>
 #include <fstream>
 #include <iomanip>
+#include <boost/format.hpp>
 
 START_NAMESPACE_STIR
 
 GeometryBlocksOnCylindrical::GeometryBlocksOnCylindrical(const Scanner& scanner)
 {
+  if (scanner.check_consistency() == Succeeded::no)
+    error("Error in GeometryBlocksOnCylindrical: scanner configuration not accepted. Please check warnings.");
```

Comment:
This is the primary compatibility change I have made. Some previously geometries may stop working. This was the case with the `test_interpolate_projdata.cxx` test and the change made to the down-sampled geometry.

It also assumes `check_consistency()` is correct.

Reply:
This is fine for me.

Reply:
as long as we put it in the release notes anyway!